core/src/mbc/mbc5.rs

```diff
@@ -27,10 +28,16 @@ pub(crate) struct Mbc5 {
 
 impl Mbc5 {
     pub(crate) fn new(data: &[u8], rom_size: RomSize, ram_size: RamSize) -> Self {
+        assert!(rom_size <= RomSize::Banks512, "More than 128 banks, but only MBC5!");
         assert!(
             rom_size.len() == data.len(),
             "Length of cartridge doesn't match length specified in ROM size header",
         );
+        assert!(
+            [RamSize::None, RamSize::Kb8, RamSize::Kb32, RamSize::Kb128].contains(&ram_size),
```

Comment:
Wouldn't it be better to whitelist the allowed sizes here? In this case we won't have to update the assertion every time the enum changes.

Reply:
This is a whitelist. `assert!` makes sure the boolean expression is `true`. So we make sure that `ram_size` is one of the listed ones. But yeah, I'm confused by asserts every time again, mostly because the messages of the assert talks about the failure case, that is: when the boolean expression is `false`.

Reply:
Of course 🙈